server/apps/api/views.py

```diff
@@ -28,6 +29,14 @@ def create(self, request, *args, **kwargs):
             return Response({"errors": (serializer.errors,)},
                             status=status.HTTP_400_BAD_REQUEST)
 
+    @staticmethod
+    def get_domain_or_create(data):
```

Comment:
Static methods in view look really weird. Use mixins or move this logic right to the view.